### ISSUE TITLE
Assignment traversing needs a special handling for ChronoBT

### ISSUE THREAD

**Issue by shnarazk** [shnarazk/splr#120]
```
$ splr -t 9000 aes.cnf
aes.cnf                                      501284,2928183 |time:  1040.09
 #conflict:    1232434, #decision:      3245711, #propagate:      886879916
  Assignment|#rem:    41050, #ass:   260696, #elm:   199538, prg%:  91.8110
      Clause|Remv:   182443, LBD2:     7638, Binc:   496190, Perm:   904711
     Restart|#BLK:     1385, #RST:     1249, trgr:        1, peak:       32
         LBD|avrg:  34.9654, trnd:   2.2807, depG:   2.9327, /dpc:     2.04
    Conflict|tASG:   1.0624, cLvl:   353.70, bLvl:   351.83, /ppc:  2035.31
        misc|proc:       19, #sub:    83161, core:        4, /cpr:  2901.83
thread 'main' panicked at 'assertion failed: 0 < path_cnt', src/solver/conflict.rs:393:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
zsh: abort      splr -t 9000 aes.cnf
```

**Comment by shnarazk** [comment]
```
$ splr -t 20000 aes.cnf                   
               
aes.cnf                                      501284,2928183 |time: 15977.84
 #conflict:   21021894, #decision:     65584197, #propagate:     5062595778
  Assignment|#rem:     8209, #ass:   245526, #elm:   247549, prg%:  98.3624
      Clause|Remv:   797993, LBD2:    30421, Binc:     9729, Perm:  1032436
     Restart|#BLK:    12141, #RST:    13168, trgr:        4, peak:       64
         LBD|avrg:   2.8254, trnd:   0.3652, depG:   4.6370, /dpc:     1.03
    Conflict|tASG:   1.0030, cLvl:     9.71, bLvl:     8.51, /ppc:    10.38
        misc|proc:       93, #sub:    38921, core:        4, /cpr:   233.58
      Result|file: ./ans_aes.cnf
s UNSATISFIABLE: aes.cnf
```

**Comment by shnarazk** [comment]
fixed by 6c3a450
